Makefile

```diff
@@ -9,14 +9,13 @@ all: fast-docker-build
 docker-build:
 	docker build --no-cache -t $(REGISTRY)/$(REPOSITORY):$(DEFAULT_TAG) .
 
-docker-build-tests: docker-build
+docker-build-tests:
```

Comment:
This is intentional for the purpose of https://github.com/fabric8-hdd/supervisor/blob/master/cico_setup.sh#L21
anyway you could instead add make docker-build below before/after https://github.com/fabric8-hdd/supervisor/blob/master/cico_setup.sh#L22 as well

Reply:
Sure.